CRM/Civicase/Angular.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+/**
+ * Option factory class for Civicase AngularJS module
+ */
+class CRM_Civicase_Angular {
+
+  private static $options = [];
+  private static $case_category_permissions;
+
+  /**
+   * @return array
+   * @throws \Exception
+   */
+  public static function getOptions(): array {
+    self::load_resources();
+    [
+      $caseCategoryId,
+      $caseCategoryName,
+    ] = CRM_Civicase_Helper_CaseUrl::getCategoryParamsFromUrl();
```

Comment:
@agileware-fj Putting this on one line fixed all the civilint errors for me (same error in another file, I'll comment there too).